packages/vuetify/src/locale/ar.ts

```diff
@@ -2,23 +2,23 @@ export default {
   badge: 'شارة',
   close: 'إغلاق',
   dataIterator: {
-    noResultsText: 'لا توجد سجلات مطابقة',
-    loadingText: 'تحميل العنصر...',
+    noResultsText: 'لم يتم إيجاد نتائج',
+    loadingText: 'يتم جلب العناصر...',
```

Comment:
تحميل العناصر. is shorter and to the point
The "..." elude that it's an ongoing operation.

Reply:
I partly agree, even though the "..." is present, the language itself can be interpreted as an action, this can easily be in a button without changing a word. Some users might get confused like, should I wait or click?

I chose wording that itself indicate that the operation is ongoing, so not to confuse any user.
What do you think?
For now, at least, I'm gonna stand by my current changes as the better translation.

Reply:
What you are saying makes sense. but the context in where this is going to be used might reduce the confusion. Also after thinking about it تنزيل would be more accurate then جلب أو تحميل. so either you choose to use I suggest using the word تنزيل.

Reply:
Your suggestion is appreciated, but allow me to disagree from personal experience this time.

The words تنزيل and تحميل are sometimes used interchangeably, true, but they still suggest slightly different meanings.

The words تنزيل refers almost exclusively to "download", while the word تحميل can be used for "load" or "download".

Using the words تنزيل is a bad call IMO, the user will think that something is being downloaded, like a file.

The way I see it, it introduces more confusion, and as for the first confusion we discussed, I used the word يتم in the sentence to clarify that this is an ongoing operation, in addition to context and "..."

All I want is the clearest translation possible, and as a regular user, incorrect wording always bugs me. So forgive me if I'm bugging you with this :)

Reply:
No worries I think what you are saying makes sense 👍